inc/footer.php

```diff
@@ -12,31 +12,44 @@
         </ul>
       </div>
     </footer>
-    <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+<script>
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
       ga('create', 'UA-47386101-5', 'auto');
       ga('send', 'pageview');
 
-    </script>
+      </script>
 
-    <script src="/js/jquery-3.1.1.min.js"></script>
-    <script src="/js/foundation.min.js?v=6.2.4"></script>
-    <script src="/js/main.js?v=<?= $asset_version ?>"></script>
-    <script src="/js/h.js?v=1"></script>
-    <script src="/js/getParameterByName.js"></script>
-    <script src="/js/khajana.js?v=<?= $asset_version ?>"></script>
+    <script src="/assets/js/vendor/jquery-3.1.1.min.js"></script>
+    <script src="/assets/js/vendor/foundation.min.js?v=6.2.4"></script>
+    <script src="/assets/js/main.js?v=<?= $asset_version ?>"></script>
+    <script src="/assets/js/h.js?v=1"></script>
+    <script src="/assets/js/getParameterByName.js"></script>
+    <script src="/assets/js/khajana.js?v=<?= $asset_version ?>"></script>
 
 <?php
-  if (is_array($scripts)) {
-    foreach ($scripts as $script) {
-      echo "    <script src=\"{$script}?v={$asset_version}\"></script>\n";
-    }
+if (is_array($scripts)) {
+  foreach ($scripts as $script) {
+    echo "    <script src=\"{$script}?v={$asset_version}\"></script>\n";
   }
+}
 ?>
+<!-- preloader -->
+<div class="hidden">
+  <script type="text/javascript">
+    <!--//--><![CDATA[//><!--
 
+      if (document.images) {
+        preborder1 = new Image();
+
+        preborder1.src = "/assets/images/border1.png";
```

Comment:
Preloading will not be necessary with the direction in which this site is going and code can be looked up again if needed. Sorry, but could you please remove and update the PR?